### DISCUSSION TITLE
TimeSeries: DataLink can't access hidden query field.

### DISCUSSION THREAD

**Discussion by Frycak** [grafana/grafana#70340] [Dashboards, Panels & Visualizations]
I am using Time Series for which data is returned from PostgreSQL query. For graph I am using some data (name; status; time), but want to use ID for Data link to external system.

Problem is that making ID hidden by using Transformation - Organise data - results that Data Link can't access ID value.

There was similar thread, but for Table (https://github.com/grafana/grafana/issues/24092).
For time series, **there is no hide in table option.**.

I have basically same issue as reported here: https://github.com/grafana/grafana/issues/59220 which is not resolved, but closed.

**Environment:**
Grafana 9.3.6
Data source: PostgreSQL

**Comment by gelicia** [comment]
ello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). I have reopened the related issue ticket. Thank you!
